set.mm

```diff
@@ -42729,13 +42746,13 @@ will do (e.g., ` O = (/) ` and ` T = { (/) } ` , see ~ 0nep0 ).
   $}
 
   ${
-    $d x A $.  $d x B $.  $d x C $.
+    $d x y A $.  $d x y B $.  $d x y C $.
     $( A member of an unordered pair of classes is one or the other of them.
        Exercise 1 of [TakeutiZaring] p. 15, generalized.  (Contributed by NM,
        13-Sep-1995.) $)
     elprg $p |- ( A e. V -> ( A e. { B , C } <-> ( A = B \/ A = C ) ) ) $=
-      ( vx cv wceq wo cpr eqeq1 orbi12d dfpr2 elab2g ) EFZBGZNCGZHABGZACGZHEABC
-      IDNAGOQPRNABJNACJKEBCLM $.
+      ( vx vy cv wceq wo cpr weq eqeq1 orbi12d dfpr2 elab2gw ) EGZBHZPCHZIFGZBH
+      ZSCHZIABHZACHZIEFABCJDEFKQTRUAPSBLPSCLMSAHTUBUAUCSABLSACLMEBCNO $.
```

Comment:
The proof is longer. So I guess something else is gained ? Axiom usage ?

Reply:
axioms yes, it's basically a continuation of #3984